format/index.md

```diff
@@ -34,20 +34,21 @@ the header `Content-Type: application/vnd.api+json`.
 Clients **MUST** ignore all parameters for the `application/vnd.api+json`
 media type received in the `Content-Type` header of response documents.
 
```

Comment:
Ok, I reordered the paragraphs a bit. Let me know what you think, and thanks always for your close readings!